composer.lock

```diff
@@ -134,72 +134,132 @@
     ],
     "packages-dev": [
         {
-            "name": "container-interop/container-interop",
-            "version": "1.2.0",
+            "name": "amphp/amp",
+            "version": "v2.6.2",
```

Comment:
I think we need #51 before merging this: that would simplify `composer.lock` changes, and prevent the failures that you observed here (incompatible dependencies)